spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -14,7 +14,8 @@ def run_qa_installer
   end
 
   def run_local_authority_installer
-    generate "qa:local"
+    generate "qa:local:files"
+    generate "qa:local:tables"
```

Comment:
The index is purely for performance.  You wouldn't notice if it does a whole table scan on a few rows.